exasol/toolbox/tools/security.py

```diff
@@ -249,7 +249,7 @@ def _row(issue):
 
 
 def security_issue_title(issue: Issue) -> str:
-    return f"🔐 {issue.cve}: {issue.coordinates}"
+    return f"{issue.cve}: {issue.coordinates}"
```

Comment:
Should likely let project-keeper folks know as this was continuing the non-ASCII symbols, so new release would helpful.